.github/workflows/ci.yml

```diff
@@ -379,38 +381,40 @@ jobs:
             mingw-w64-x86_64-cmake
             mingw-w64-x86_64-cmocka
             mingw-w64-x86_64-ninja
-            mingw-w64-x86_64-libxml2
-            mingw-w64-x86_64-pugixml
-            mingw-w64-x86_64-libjpeg-turbo
-            mingw-w64-x86_64-zlib
+            mingw-w64-x86_64-nsis
+            mingw-w64-x86_64-dbus-glib
+            mingw-w64-x86_64-drmingw
             mingw-w64-x86_64-exiv2
+            mingw-w64-x86_64-flickcurl
```

Comment:
can be removed as it is not used anymore. we do not support flickr or any other external library service now except the Open Source Piwigo.